Dockerfile-debian.template

```diff
@@ -2,7 +2,9 @@
 FROM debian:jessie
 
 # explicitly set user/group IDs
-RUN groupadd -r postgres --gid=999 && useradd -r -g postgres --uid=999 postgres
+RUN groupadd -r postgres --gid=999 \
+  && useradd -r -g postgres --uid=999 postgres \
+  && install -d -m 0700 -o postgres -g postgres /home/postgres
```

Comment:
I thought this was going to go in the barman/Dockerfile.template like you had originally done and not here so that the /home/postgres directory doesn't exist when not using barman.